yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ShareSessionReceipt.java

```diff
@@ -0,0 +1,206 @@
+package com.yoti.api.client.spi.remote.call;
+
+import java.util.Base64;
+import java.util.Optional;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(builder = ShareSessionReceipt.Builder.class)
+public final class ShareSessionReceipt {
+
+    private final String id;
+    private final String sessionId;
+    private final String timestamp;
+    private final Content content;
+    private final Content otherPartyContent;
+    private final String wrappedItemKeyId;
+    private final byte[] rememberMeId;
+    private final byte[] parentRememberMeId;
+    private final byte[] wrappedKey;
+
+    private ShareSessionReceipt(Builder builder) {
+        id = builder.id;
+        sessionId = builder.sessionId;
+        timestamp = builder.timestamp;
+        content = builder.content;
+        otherPartyContent = builder.otherPartyContent;
+        wrappedItemKeyId = builder.wrappedItemKeyId;
+        rememberMeId = builder.rememberMeId;
+        parentRememberMeId = builder.parentRememberMeId;
+        wrappedKey = builder.wrappedKey;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getSessionId() {
+        return sessionId;
+    }
+
+    public String getTimestamp() {
+        return timestamp;
+    }
+
+    public byte[] getProfile() {
+        return content.profile();
+    }
+
+    public byte[] getExtraData() {
+        return content.extraData();
+    }
+
+    public byte[] getOtherPartyProfile() {
+        return otherPartyContent.profile();
+    }
+
+    public byte[] getOtherPartyExtraData() {
+        return otherPartyContent.extraData();
+    }
```

Comment:
As mentioned above, all of these _might_ not exist in a receipt. Make sure that this doesn't blow up if any of these are `null` in the response.